payload.go

```diff
@@ -0,0 +1,83 @@
+package wappin
+
+import (
+	"github.com/fairyhunter13/phone"
+	"github.com/flip-id/valuefirst/manager"
+)
+
+const (
+	// TokenBearer is the bearer token for Wappin.
+	TokenBearer = "Bearer "
+)
+
+// ResponseMessage is a response message from the Wappin.
+type ResponseMessage struct {
+	MessageID      string                 `json:"message_id"`
+	Status         string                 `json:"status"`
+	Message        string                 `json:"message"`
+	Data           map[string]interface{} `json:"data"`
+	HttpStatusCode int                    `json:"-"`
+	RawData        string                 `json:"-"`
+}
+
+// RequestWhatsappMessage is a request message for sending WhatsApp message.
+type RequestWhatsappMessage struct {
+	ClientID        string            `json:"client_id"`
+	ProjectID       string            `json:"project_id"`
+	Type            string            `json:"type"`
+	RecipientNumber string            `json:"recipient_number"`
+	Params          map[string]string `json:"params"`
+	Token           string            `json:"-"`
+}
+
+// Default returns the default request for send Whatsapp message in Wappin.
+func (r *RequestWhatsappMessage) Default(o *Option) *RequestWhatsappMessage {
```

Comment:
```suggestion
func (r *RequestWhatsappMessage) Default(o *Option) *RequestWhatsappMessage {
func (r *RequestWhatsappMessage) Default(o *Option)  {
```
Omit the return value should be OK I guess since the pointer receiver is the same struct as the return value.